app/views/dashboard.py

```diff
@@ -9,7 +9,7 @@
 from datetime import datetime
 
 from app.util import MobileSupportedView, most_recent_game
-from app.models import Game, Tag, Player, PlayerRole, SignupLocation, Legacy, SupplyCode, User
+from app.models import Game, Tag, Player, PlayerRole, SignupLocation, Legacy, SupplyCode, User, TagType, Email, EmailRule
```

Comment:
You need to add all these things to the `__init__.py` files in the models folder. Otherwise they will not build.